### ISSUE TITLE
raspberry pi and alexa avs sample,wakeWordAgent problem

### ISSUE THREAD

**Issue by thrylosthyra7** [alexa/alexa-avs-sample-app#552]
ERROR:Failed to open PortAudio stream.Invalid sample rate
ERROR:KittAiSnowboyWakeWordEngine: Initialization error:Failed to open PortAudio stream.
ERROR:WakeWordAgent: exception in constructor: Failed to open PortAudio stream.
ERROR:main: Exception happened: Failed to open PortAudio stream.
INFO:Shutting down Wake Word Agent

does anyone really solved it?none of the rec t.wav or rec temp.wav commands work for me,but the mic (and alexa) works perfectly with the "listen" button.

I have also tried this solution:
install this:sudo apt-get install python-pyaudio python3-pyaudio sox
and change asoundrc file to this:

pcm.!default {
  type asym
   playback.pcm {
     type plug
     slave.pcm "hw:0,0"
   }
   capture.pcm {
     type plug
     slave.pcm "hw:1,0"
   }
}

any ideas would be really much appreciated..

**Comment by Alexnagel** [comment]
You could check #533 to see if one of those options works for you

**Comment by thrylosthyra7** [comment]
no,it doesn't work for me!I reinstalled but it seems like stop working after some hours of use.

**Comment by yugoren** [comment]
Hi @thrylosthyra7,

Just to double-check on your progress, a fresh install has solved the initial issue but now you're getting it during continuous use?

**Comment by thrylosthyra7** [comment]
Exactly!!when it happens,it says the same errors as before..

Στις 22 Μαρ 2017 8:04 μ.μ., ο χρήστης "Yusuf Goren" <
notifications@github.com> έγραψε:

**Comment by namanmittal-namanm** [comment]
HI @thrylosthyra7 can you please paste the output of arecord -l and arecord -L.
